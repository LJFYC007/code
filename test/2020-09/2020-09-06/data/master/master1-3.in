7 999999998 999999997
754988150 16665527 901309463 868431603
210493635 666277040 962869656 711197101
3272656 207091753 102229696 382552504
326580449 57498942 691451290 129475896
376595869 287952863 483548390 742246669
496072481 493061448 850417630 974382603
114084533 547029018 597704153 892789814
