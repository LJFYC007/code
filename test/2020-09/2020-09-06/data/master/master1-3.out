17447568374978260
